src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static final int number = 4;

```

Comment:
The constant `number` should have a more descriptive name, such as `NUMBER_OF_FIGURES`, to clearly indicate its purpose.